### PR TITLE
report: introduce Roboto variable font

### PR DESCRIPTION
**Pull request by paulirish**
i dont love doing a network request to google webfonts for roboto.
and datauri'ing all our 3 roboto weights would be lots of bytes. so ... variable fonts the the rescue!

https://github.com/TypeNetwork/Roboto-Delta has a variable font that Google commissioned. (i used v2). There was an [earlier version](https://github.com/TypeNetwork/Roboto), too, but it's bigger.

I [compressed](https://github.com/google/woff2) as woff2 and am including as a webfont. the `  font-weight: 1 999;` in the @font-face rule allows all css to just pull various font-weights from it, depending on what they want. so normal/500/bold work, as do any integers 1 through 999. ;)

**Filesize**: it's 135kb extra in the CSS. :/

**Future work**: There's still an opportunity to subset the font and remove unneeded tables. It's possible it could come down to ~30kb. see also [How to use variable fonts in the real world](https://medium.com/clear-left-thinking/how-to-use-variable-fonts-in-the-real-world-e6d73065a604) and [Munter/subfont: Command line tool to optimize your webfont loading.](https://github.com/Munter/subfont).

This all works in FF and Safari as well.
#8185

@yuinchien can you take a look?

### CODE REVIEW

File: lighthouse-core/report/html/report-styles.css [hunk 1]
```diff
@@ -1341,3 +1341,9 @@
...
...Gv+tQdrbN8Gy//MJ9n2U51GkRIHJn0VL6Dn9AwXQhfnGVZaNLZij+x2wVhbvVZJBvRDHXCbUy5tDWGqDIdZaOvDvLgQ4zD9/4lB1h0rxEdcZYcun/3EPgr2qjOncvxJM3mgPvf+yDJM2GypIEq9PhLNnUkU5460na6pTqXdNqrT9sB0TWp/RoBHykQW8yi/vNFjDzLtw8aMfnH9+rLDCoeO2PXKDqc2PKPrSDqXuAQwFChPeStoty10tBMSGssqUHsci97IrZU0+SzWXfWuqC/GkOBG7/N4XGcCCvOo64j7u9LANvdakHOO6inIez9cWPNi2+Xr+uNMJCeoznl0XEfTqrtqSSyiJBXNKyDJUZX3h3VqyYuHOes0xCVXEs7+fAemsWfwkwUNnFtrA2PdvFNrUmgxecpCN26DbC0f7dYydnZvNDaU3B9EpSVE37AkxV5AycmpycHTreY8k7ExxlgRzXeWJ8pl67pT2dkGodKZEnV4vt4rvD7kVlXS3eHqKPZ1lWQXjFeHY3kkO2HJXKrNZNk30GS5MO4rN6i6RtCyfLtiKaTlew4K9ius3CF1eHlWK3hFhuabeULqcp4qi9RZUipO1R8I/dd3JLdjEjMSvo5B8m6RX84efsbHYhfoaVdH+pUK7fuSeU/qi180vY6jqUJ/PAPNJi2Hkh4KMGN2XOeR5Y5Zf99TPB7eLLxmgL3f1YembDnMzsLF9QEszvZC6AqYO+Wemk9HW0uk2USVnqwqBrqXA83wR3zW+uUpBMawQtEQTWvJjcmLU6HVr4uN9BWE+VZTrneuWsqqbA+JQeVIyEaa2ZsU5WGlpB0sg4W03SJNbipw2ajzeiGk9SBL5+rLQiiTyI3jAZ8D1Mxfo4xE7v86JuQuzkGUt28mGH5FPFsAxT5yUFh+wdCt5U16/Q6mqhblTLhkZl6mBK2xj/QpXri74MqugJXOXpcL6UNKiULOWD/r5gb0Ed2INCcP6jAmxFQKwJYckJyi0c4U9+izkVyjLH8ab9SivliNcoxR3JitIni/vh3cp6eVpW/mie3YwkZizxR/CdGryx1JfRjK9WB0bwKsBKqBkvPs4y/s4Deb199ma7VZGLvoMtdg4mGqFNPT9ct4Akg+0zRvy7o8w0QAc1cYyEoHxUD8UvHEEQH3txuO4KFiEoiNcYJeRtmWilqRRSQGSEcdwkrExdN/1
```
Comment (Hoten) [inline lighthouse-core/report/html/report-styles.css]: RIP scrollbar :(

Comment (Hoten) [reply lighthouse-core/report/html/report-styles.css]: another candidate for an enriched asset pipeline (like #8632)

**Comment by yuinchien** [comment]
@paulirish @Hoten seems like the beta versions of Roboto VF might have incomplete glyph set and could render bad results. Is it possible to hold the VF release until it's production ready? I believe Google Fonts is aiming sometime soon, getting timeline from them, Paul i've added you to the email thread.

File: lighthouse-core/report/html/report-styles.css (hunk 1 above)
Comment (patrickhulce) [reply lighthouse-core/report/html/report-styles.css]: yeah +1 @Hoten's approach is suddenly much more necessary :)

**Comment by brendankenny** [comment]
> Filesize: it's 135kb extra in the CSS. :/

and 102KB in the bundle (and viewer) after gzip...

**Comment by brendankenny** [comment]
counter-proposal: we live with a web font or just don't use it :)

135KB is big, and in the end we're talking about a (semi-)portable self-contained report format. If we want to drop those requirements we can start doing a lot of things with associated files. Meanwhile we're soon going to be including schema.org data, giant screenshots...things are going to get big fast

> there's still an opportunity to subset the font and remove unneeded tables. It's possible it could come down to ~30kb

how unneeded are they with i18n? And there's more maintenance and fiddliness...

**Comment by brendankenny** [comment]
we're not making a move on this any time soon. Closing, but leaving the branch open

**Pull request closed without merging**